tst/testinstall/opers/HallSubgroup.tst

```diff
@@ -0,0 +1,20 @@
+gap> START_TEST("HallSubgroup.tst");
+gap> G := GL(3,4);;
+gap> HallSubgroup(G, [2,3]);
+fail
+gap> G := PSL(4,2);;
+gap> HallSubgroup(G, [2,3]);
+<permutation group of size 576 with 8 generators>
```

Comment:
There might be problem running this test with different packages loaded, because the presentation of the answer might change. Maybe ask for an ````IdGroup````, or something similar?